### PR TITLE
6 create team page

### LINKED ISSUE #6: Team Page

### PR DESCRIPTION
**Pull request by Ram-Raghav-S**
The team page at the moment is made to work for large screens. It does have problems with scaling the text for medium and small devices - this will be fixed in a separate issue. 

Do let me know if any changes are required.

### CODE REVIEW

**Review by PierreLessard — DISMISSED** [review]
There was a rounding utility class being used on the team card div causing the corners to look a little cut off compared to the figma so I removed it and also added some img alts to get rid of the warnings. Otherwise, really nice! approved

**Comment by Ram-Raghav-S** [comment]
@PierreLessard I added the rounding in my original code because leaving it out caused an issue with containing certain images. The images in the original PR did not show this issue, but I have now added another image (the cat) to the list for which this issue shows up. You can try to remove the rounding from my code and test it out.

I have tried to add the rounding back with a different radius, but I agree that it still looks cut off compared to the figma. 

I think we will have to find another way of getting around this issue other than removing the rounding. Otherwise, it is also possible to preprocess images before adding them to the page.

Let me know what you think is best here.

**Comment by PierreLessard** [comment]
I fixed the corner issue by going into figma and creating a subtracted shape svg for that corner and laid it above the image. I had to add a few pixels below as well to prevent the image clipping below during graphical issues. @Ram-Raghav-S I want to get the ok from you that you think this method works before I approve it.

**Comment by Ram-Raghav-S** [comment]
@PierreLessard It looks great on a large screen but it causes problems when we resize the screen. Would we be to do something that makes it looks good on small screens as well in the future? If yes, I think this is ok.

@takeachangs what do you think about this?

**Review by PierreLessard — APPROVED** [review]
Seongmin just let me know to worry about mobile responsiveness when we are done the rest. Also he said that most images will be cropped in a way that we dont have to worry about that corner.

File: frontend/src/components/TeamTile.js
Comment (takeachangs) [inline frontend/src/components/TeamTile.js:1]: Good on desktop view, but resizing for mobile view is a little wonky. Don't forget to check in mobile view too for future edits!

**Review by takeachangs — APPROVED** [review]
Great that you created a separate data file!

**Pull request merged**
